libs/sdk-core/src/persist/transactions.rs

```diff
@@ -46,6 +50,12 @@ impl SqliteStorage {
         Ok(())
     }
 
+    fn delete_pending_payments(&self) -> Result<usize> {
+        self.get_connection()?
+            .execute("DELETE FROM payments WHERE pending = true", [])
```

Comment:
Need to make sure we only delete lightning payments. Pending can be true also for closed channel.

Reply:
Done, tested: e7f9d6ab472baf3f6246d342ee0bfa7a426c881d